fcrepo-kernel-impl/src/main/java/org/fcrepo/kernel/impl/services/ManagedPropertiesServiceImpl.java

```diff
@@ -45,7 +46,7 @@
     @Override
     public Stream<Triple> get(final FedoraResource resource) {
         final List<Triple> triples = new ArrayList<>();
-        final var subject = createURI(resource.getId());
+        final var subject = createURI(FedoraResourceIdConverter.resolveFedoraId(resource));
```

Comment:
That's an excellent question... I don't know. I just changed it there because it seemed to have an affect on one of the requests I testing. Perhaps @dbernstein or @whikloj can say?

Reply:
HttpResourceConverter is deprecated and I really wondered whether we needed a new class or the whole `resolveFedoraId` logic could not be incorporated into the FedoraResource class itself?